ui/service.go

```diff
@@ -281,40 +282,51 @@ func (gs *uiService) ListChannels(ctx context.Context, token string) ([]byte, er
 }
 
 func (gs *uiService) RemoveChannel(ctx context.Context, token, id string) ([]byte, error) {
-	err := gs.sdk.DeleteChannel(id, "123")
+	err := gs.sdk.DeleteChannel(id, gs.token)
 	if err != nil {
 		return []byte{}, err
 	}
-	return gs.ListChannels(ctx, "123")
+	return gs.ListChannels(ctx, gs.token)
 }
 
 func (gs *uiService) CreateGroups(ctx context.Context, token string, groups ...sdk.Group) ([]byte, error) {
+
 	for i := range groups {
-		_, err := gs.sdk.CreateGroup(groups[i], "123")
+		_, err := gs.sdk.CreateGroup(groups[i], gs.token)
 		if err != nil {
 			return []byte{}, err
 		}
 	}
-	return gs.ListGroups(ctx, "123")
+
+	return gs.ListGroups(ctx, gs.token)
 }
 
 func (gs *uiService) ListGroups(ctx context.Context, token string) ([]byte, error) {
-	tpl, err := template.ParseGlob(templateDir + "/*")
+	tpl := template.New("groups")
+	tpl = tpl.Funcs(template.FuncMap{
+		"toJSON": func(data map[string]interface{}) string {
+			ret, _ := json.Marshal(data)
+			return string(ret)
+		},
+	})
+	var err error
+
+	tpl, err = tpl.ParseGlob(templateDir + "/*")
 	if err != nil {
 		return []byte{}, err
 	}
 
-	grpsPage, err := gs.sdk.Groups(0, 100, "123")
+	thsPage, err := gs.sdk.Groups(0, 100, gs.token)
 	if err != nil {
 		return []byte{}, err
 	}
 
 	data := struct {
 		NavbarActive string
-		Groups       []auth.Group
+		Groups       []sdk.Group
 	}{
 		"groups",
-		grpsPage.Groups,
+		thsPage.Groups,
```

Comment:
CI is failling, `thsPage.Groups` is of type `[]auth.Group` not `[]sdk.Group`

Reply:
It is connected with previous PR. It will be fine now, but i will change tshPage to grpsPage :)